packages/tenacity-git/tenacity-git.pacscript

```diff
@@ -0,0 +1,111 @@
+name="tenacity-git"
+url="https://github.com/tenacityteam/tenacity.git"
+build_depends="build-essential cmake git python3-pip ffmpeg"
+depends="libgtk2.0-dev libasound2-dev libavformat-dev libjack-jackd2-dev uuid-dev Qt5Widgets libqt5gui5"
+replace="audacity"
+description="FLOSS Audacity Fork. No telemetry, crash reports and other shenanigans like that!"
+optdepends="ffmpeg: additional import/export capabilities"
+removescript="yes"
+maintainer="echometer <guacamolek@yandex.com>"
+
+pkgver() {
+    git ls-remote "${url}" master | cut -f1 
+}
+version="$(pkgver)"
+
+prepare() {
+    sudo pip3 install conan
+}
+
+build() {
+    sudo mkdir build
+    cd build
+    sudo cmake -G "Unix Makefiles" \
```

Comment:
The script is fine but is `sudo` finally required?

When I run scripts like ![opendoas-git](https://github.com/pacstall/pacstall-programs/pull/95) on my machine it works as intended, but when you check the CI logs it says permission denied etc. So what's going on?

Reply:
the CI is on the master branch, the logname fix is still on the develop branch

Reply:
> If you plan to merge this one in the near future, add it.
If not, keep it the way it is currently